2023-risks/m4-inadequate-privacy-controls.md

```diff
@@ -0,0 +1,79 @@
+---
+layout: col-sidebar
+title: "M4: Inadequate Privacy Controls"
+---
+
+# Threat Agents
+
+**Application Specific**
+
+Privacy controls are concerned with protecting Personally Identifiable Information (PII). This information is valuable to attackers for various reasons, e.g., for
+
+- impersonating the victims in cases of fraud,
+- misusing the victims' payment data, or
+- blackmailing the victims with sensitive information.
+
+# Attack Vectors	
+
+**Exploitability MEDIUM**
+
+Typical sources for PII are either protected or hard to access, e.g., the sandbox of the app, the network communication with the server, the app's logs and backups. Some are easier to observe, like URL query parameters and clipboard content.
+
+Obtaining PII thus requires the attacker to breach security on another level. He could eavesdrop on the network communication, access file system, clipboard or logs with a trojan or get his hands on the mobile device and create a backup to analyse.
+
+# Security Weakness	
+
+**Prevalence COMMON** <br/>
+**Detectability EASY**
+
+Risks of privacy violations increase due to careless handling of PII by developers. PII should always be processed with the posibility in mind that an attacker could access communication and storage media. Hence, PII should not be stored or transfered
```

Comment:
Risks of privacy violations increase due to careless handling of PII by developers. PII should always be processed with the possibility in mind that an attacker could access communication and storage media. Hence, PII should not be stored or transferred
unless absolutely necessary. If it is to store or transfer, one should consider encrypting or anonymizing it if feasible. Leakage of PII to uncontrollable sinks, e.g., logs, clipboard, URL-parameters, must be prevented.

An attacker could manipulate or abuse the data collected by the mobile app, potentially leading to privacy infringements or identity theft.